### ISSUE TITLE
关于semi半监督模式下的测试精度计算问题

### ISSUE THREAD

**Issue by nmvbxcz** [eecn/Hyperspectral-Classification#13]
你好，想请教一下
` elif supervision == 'semi':

                outs = net(data)

                output, rec = outs

                #target = target - 1

                loss = criterion[0](output, target) + net.aux_loss_weight * criterion[1](rec, data)`

这里返回的rec对应的是模型中分类前一步结果送入nn.Linear(self.features_sizes, input_channels)中得到，这里对于模型
`criterion = (nn.CrossEntropyLoss(weight=kwargs['weights']), lambda rec, data: F.mse_loss(rec, data[:,:,:,patch_size//2,patch_size//2].squeeze()))`这段有什么作用？

**Comment by rongtongxueya** [comment]
我也想知道
